md2conf/__init__.py

```diff
@@ -5,7 +5,7 @@
 Confluence API endpoints to upload images and content.
 """
 
-__version__ = "0.1.13"
+__version__ = "0.1.14"
```

Comment:
Please don't increment the version number yet. Several new features have been added recently. This is likely to be part of `0.2.0`.